OWNERS

```diff
@@ -9,7 +9,7 @@ approvers:
   - heartlock
   - mrunalp
   - saschagrunert
-  - timstclair
```

Comment:
I think this is meant to be @tallclair - https://github.com/timstclair has a pointer to the newer GH handle.

Reply:
Ah, good point ... the other Time St. Clair :) I've updated the commit with the correct GitHub handle.